app/helpers/application_helper.rb

```diff
@@ -6,4 +6,8 @@ def page_title(title)
   def claim_in_progress?
     session.key?(:claim_id)
   end
+
+  def support_email_address
+    "additionalteachingpayment@digital.education.gov.uk"
+  end
```

Comment:
In here rather than the i18n file?

Reply:
I hadn't considered using the locale file for this.. I guess it was partly in anticipation of the method having to return a different support email under different circumstances (i.e. for the two different policies), but perhaps that's a touch of YAGNI. Should I just use the locale file instead?

Reply:
That's my hunch. It's what we're doing for e.g. journey name, so it would be consistent.

Reply:
ok, I've updated it to load from the locale file instead